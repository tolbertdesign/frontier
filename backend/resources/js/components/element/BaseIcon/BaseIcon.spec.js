import { shallowMount, createLocalVue } from '@vue/test-utils'
import BaseIcon from './BaseIcon.vue'

const localVue = createLocalVue()

const name = 'name'
const width = 'width'
const height = 'height'

describe('BaseIcon', () => {
  it('Snapshop test for BaseIcon', () => {
    const wrapper = shallowMount(BaseIcon, {
      localVue,
      propsData: {
        name: name,
        width: width,
        height: height,
      },
    })
    expect(wrapper).toMatchSnapshot()
  })
})
