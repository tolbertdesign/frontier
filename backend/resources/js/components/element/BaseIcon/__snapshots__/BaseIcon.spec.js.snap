// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseIcon Snapshop test for BaseIcon 1`] = `
<div class="icon-wrapper"><svg width="width" height="height" class="icon">
    <use xlink:href="/v3-assets/dashboard/icons.svg#name"></use>
  </svg> </div>
`;
